src/app-pages/post/PostDetailPage.tsx

```diff
@@ -68,26 +68,86 @@ export default function PostDetailPage({ postId }: PostDetailPageProps) {
     setLikedUsersOpen(true);
   };
 
+  const handleDelete = async () => {
+    try {
+      await deletePost(numericPostId);
+
+      alert('게시글이 삭제되었습니다.');
+      setShowDeleteAlert(false);
+      router.push(`/board/${post.boardId}`);
```

Comment:
게시글 삭제하고 나서, 기존에 있던 경로로 돌아가는 게 자연스러울 것 같은데 어떨까요?
예를 들면, 내 게시글 목록에서 내 게시글에 들어가서 삭제했을 때는 내 게시글 목록으로 돌아가고,
게시판 목록에서 내 게시글에 들어가서 삭제했을 때는 게시판 목록으로 돌아가는 걸로요!
```suggestion
      alert('게시글이 삭제되었습니다.');
      setShowDeleteAlert(false);
      router.back();
```

Reply:
기존 코드 테스트해봤을 때, 게시판 url이 `/board?boardId=1&category=all` 형태라 그런지
`/board/${post.boardId}` 로 가면 오류가 나는 것 같습니다!

Reply:
좋은 것 같습니다! boardpage에서 boardId만 폴더로 빼고 category만 쿼리 유지하도록 하겠습니다

Reply:
내 게시글 목록에서 내 게시글에 들어가서 삭제했을 때는 내 게시글 목록으로 돌아가는 로직을 생각하지 못했네요! router.back()을 사용하도록 수정해두었습니다!